### PR TITLE
Corrected error caused by TCS 128 merge and added logging

### PR DESCRIPTION
**Pull request by ThomasGillespie**
Fixed error from merge on Friday

### CODE REVIEW

**Review by SeanW17 — COMMENTED** [review]
Could you fix this log message so its indented consistently with others. Or vice versa.

**Comment by ThomasGillespie** [comment]
> Could you fix this log message so its indented consistently with others. Or vice versa.

The reason the previous log message is indented more is because it is inside of an If statement

**Comment by SeanW17** [comment]
And line 34-38?

**Review by jonathonmcmurray — APPROVED** [review]

**Review by emansanti — APPROVED** [review]

File: code/segmentedtickerplant/datastripe.q [hunk 1]
```diff
@@ -28,6 +28,8 @@ subsegment:{[tbl;segid];
      if[default~"all";suballtabs: .stpps.t except stripedtables;
        if[tbl in suballtabs;
           .lg.o[`sub;m:"Table ",string[tbl]," is to be subscribed unfiltered for segment ",string[segid],""]]];
+     if[tbl in stripedtables;
+       .lg.o[`sub;"Table ",string[tbl]," is to be subscribed filtered for segment ",string[segid],""]]
```
Comment (jonathonmcmurray) [inline code/segmentedtickerplant/datastripe.q:32]:
```suggestion
       .lg.o[`sub;"Table ",string[tbl]," is to be subscribed filtered for segment ",string[segid],""]];
```

missing semi-colon

**Pull request merged**